app/routes/reports.tsx

```diff
@@ -16,23 +16,48 @@ export const meta: MetaFunction = () => {
 	];
 };
 
+interface IPageData {
+	reports: Report[];
+	numOfContributors: number;
+}
+
 export const loader: LoaderFunction = async () => {
-	try {
-		const reports = await fetchReports();
-		const numOfContributors = await getNumberOfContributors();
+	const reports = await fetchReports();
+	const numOfContributors = await getNumberOfContributors();
+
+	return {
+		reports,
+		numOfContributors,
+	};
+};
```

Comment:
why don't we want to keep the error handing here?

Reply:
we do! thanks for reminding me :), good catch

Reply:
👍 ok i thought maybe `safeReports` was meant to replace the error handling. seemed... less safe 😆

Reply:
Oh no lol, re: safeReports: https://github.com/VoiceDeck/app/pull/18#discussion_r1499563181